### PR TITLE
[ContactStructuralMehcanicsApplication] Second analysis update

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

File: applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis.py [hunk 1]
```diff
@@ -50,18 +50,18 @@ def __init__(self, model, project_parameters):
             KratosMultiphysics.Logger.PrintWarning("StructuralMechanicsAnalysis", warn_msg)
 
         if not solver_settings.Has("time_stepping"):
-            KratosMultiphysics.Logger.PrintInfo("StructuralMechanicsAnalysis", "Using the old way to pass the time_step, this will be removed!")
+            KratosMultiphysics.Logger.PrintWarning("StructuralMechanicsAnalysis", "Using the old way to pass the time_step, this will be removed!")
```
Comment (loumalouomega) [inline applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis.py:53]: @philbucher Turned this into warnings

Comment (philbucher) [reply applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis.py:53]: :+1:

**Review by philbucher — APPROVED** [review]
Thanks!

**Comment by philbucher** [comment]
I did not run the tests but I didn't see any wrong changes

**Pull request merged**